project/src/com/google/daggerquery/example/modules/AccommodationModule.java

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package com.google.daggerquery.example.modules;
+
+import com.google.daggerquery.example.models.Apartment;
+import com.google.daggerquery.example.models.Villa;
+import dagger.Module;
+import dagger.Provides;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+@Module
+public class AccommodationModule {
+  @Provides
```

Comment:
I think you need `@ElementsIntoSet` to make this a `Set` multibinding collection. (see https://dagger.dev/dev-guide/multibindings.html#declaring-multibindings)

Reply:
Never mind, it shouldn't make any difference to the demo.